doc/cygnus-common/installation_and_administration_guide/management_interface_v1.md

```diff
@@ -41,6 +41,11 @@ Content:
     * [GET `/v1/admin/metrics`](#section7.1)
     * [DELETE `/v1/admin/metrics`](#section7.2)
 * [Available aliases](#section8)
+* [Name Mappings](#section9)
+    * [GET `/v1/namemappings`](#section9.1)
+    * [POST `/v1/namemappings`](#section9.2)
+    * [PUT `/v1/namemappings`](#section9.3)
+    * [DELETE `/v1/namemappings`](#section9.4)
 
 ## <a name="section1"></a>Apiary version of this document
 This API specification can be checked at [Apiary](http://telefonicaid.github.io/fiware-cygnus/api/latests) as well.
```

Comment:
http://telefonicaid.github.io/fiware-cygnus/api/latests result in 404. Maybe the link is obsolete.

Reply:
fixed in dfc6fbaf49b9cf1af501cc2833dce6e7aa6a76ab